report/src/system_evaluation/results.tex

```diff
@@ -32,28 +32,40 @@ \section{System evaluation}\label{sec:experiment_results}
+    \multicolumn{1}{|l|}{}        & \multicolumn{1}{c|}{a}     & \multicolumn{1}{c|}{b}    & \multicolumn{1}{c|}{c}     & \multicolumn{1}{c|}{d}     & \multicolumn{1}{c|}{e}     & \multicolumn{1}{c|}{f}     & \multicolumn{1}{c|}{g}     & \multicolumn{1}{c|}{h}    & \multicolumn{1}{c|}{i}    & \multicolumn{1}{c|}{j}    & \multicolumn{1}{c|}{k}    & \multicolumn{1}{c|}{l} \\ \hline
+    2                             & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{8/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{9/10}  & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{0/10} & \multicolumn{1}{l|}{4/10} & \multicolumn{1}{l|}{5/10} & \multicolumn{1}{l|}{7/10} & 8/10                   \\ \hline
+    3                             & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{7/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{5/10} & \multicolumn{1}{l|}{5/10} & \multicolumn{1}{l|}{5/10} & \multicolumn{1}{l|}{8/10} & 8/10                   \\ \hline
+    4                             & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{9/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{10/10} & \multicolumn{1}{l|}{6/10} & \multicolumn{1}{l|}{6/10} & \multicolumn{1}{l|}{7/10} & \multicolumn{1}{l|}{9/10} & 10/10                  \\ \hline
+    \end{tabular}}
+    \caption{Evaluation results from Lenovo TabM10 FHD Plus showing the number of correct classifications out of a possible ten for each $k$-value}
+    \label{lst:resultsTablet}
+\end{table}
 
-To create an environment that more accurately represents a meeting room in use, several devices were used for testing. 
-The method for collecting results from these tests mirrored previous ones, by making $10$ classifications on both a Samsung Galaxy A53, a Samsung Galaxy S20 phone, and a Lenovo TabM10 FHD Plus tablet. 
-The classification results were gathered in the locations shown on Figure \ref{fig:experiment_room}
-During the experiment, k-values of $2$, $3$, and $4$ were used.  
-The results of the experiment can be seen in Tables \ref{lst:resultsPhone}, \ref{lst:resultsPhoneS20}, and  \ref{lst:resultsTablet}.
-Table \ref{lst:resultsPhone_precision} summarizes the experiment's results, indicating the percentage of true positive classifications. 
+On Table \ref{lst:resultsPhone_precision} see that a lower $k$-value results in a larger uncertainty in the classification.
+However, it is worth noticing that for measuring points \textit{B} and \textit{J}  (see Figure \ref{fig:experiment_setup}) a large uncertainty in classification is observed.
+We suspect that this is a consequence of the glass pane next to the door having a much lower absorbation of radio frequencies than the door and the wall next to it. 
```

Comment:
Need cite?
That's a nice claim. How about you back it up with a source
```suggestion
We suspect that this is caused by the glass pane next to the door having a much lower absorbation of radio frequencies than the door and the wall next to it. 
```

Reply:
That is why I have written suspect. I have not been able to find any paper describing the effect of glass vs regular wall, or just glass for that matter.

Reply:
Suspect doesn't, when I read the sentence, imply our belief that glass has lower absorption of radio frequencies, but rather that we suspect the low absorption is the cause - which we are not able to provide evidence for.